doc/announce/changelog-only.rst

```diff
@@ -0,0 +1,11 @@
+Less announcing, more change-logging
+------------------------------------
+
+With version 2.5.0 we dropped creating special announcement documents and rely on communicating
+all relevant changes through the
+`CHANGELOG <https://github.com/tox-dev/tox/blob/master/CHANGELOG>`_. See at
+`pypi <https://pypi.org/project/tox/>`_ for a rendered version of the last changes containing
+links to the important issues and pull requests that were integrated into the release.
+
```

Comment:
When we really get rid of them I would add the following also:

The historic release announcements are still online here:

* http://tox.readthedocs.io/en/2.4.0/announce/release-0.5.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.0.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.1.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.2.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.3.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.4.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.4.3.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.8.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-1.9.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-2.0.html
* http://tox.readthedocs.io/en/2.4.0/announce/release-2.4.0.html